protobuf/log_extractor.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto2";
+
+package log_extractor;
+
+message LogEvent {
+  required uint64 timestamp = 1;
+  required string category = 2; // e.g., "validation", "net", etc.
+  oneof event {
+    UnknownLogMessage unknown_log_message = 3;
+    BlockConnectedLog block_connected_log = 4;
+  }
+}
```

Comment:
What do you think about storing the "raw message" for known logs? Or when we gonna implement some known log, we get everything that is a variable in that string? I ask this because btc-core could change messages and maybe its interesting to store what it was at that moment🤔 

Also this is a mistery to me, on how to organize the LogMatcher to support many versions of btc-core, maybe many "case" matches from different versions of that log there, idk.